runner/local/simple.go

```diff
@@ -88,47 +79,57 @@ func (r *simpleRunner) StatusAll() ([]runner.ProcessStatus, error) {
 	return statuses, nil
 }
 
-func (r *simpleRunner) Abort(run runner.RunId) (runner.ProcessStatus, error) {
-	return r.updateStatus(runner.AbortStatus(run))
+func (r *simpleRunner) Abort(runId runner.RunId) (runner.ProcessStatus, error) {
```

Comment:
Ugh, we've done fooId everywhere up til now, but I think Go style says to do fooID. Not sure if we should be consistent, or start doing the right thing and go back and change at some point... WDYT?